docs/Tutorials - start here/002_Uploading_data_and_checkpoints_to_s3.md

```diff
@@ -0,0 +1,90 @@
+# Uploading data and model checkpoints to s3
+There are two ways to upload them: 
+1. using python script
+2. from cli (command line interface)
+
+Both optinions will be considered.
+
+For both uploading data and model using either cli or python script you can omit access_key and secret_key arguments, but they will be asked. So you need to specify them anyway.
```

Comment:
"When uploading data or a model using either the command line interface (CLI) or an API, you have the option to omit the access_key and secret_key arguments. The framework will search for these values in the `~/.aws/credentials` and `.env` files. However, if these values are not found, you will be prompted in CLI."